src/main/java/com/arangodb/model/DatabaseUsersOptions.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * DISCLAIMER
+ *
+ * Copyright 2016 ArangoDB GmbH, Cologne, Germany
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright holder is ArangoDB GmbH, Cologne, Germany
+ */
+
+package com.arangodb.model;
+
+import java.util.Map;
+
+/**
+ * @author Michele Rastelli
+ */
+public class DatabaseUsersOptions {
+
+    private String username;
+    private Map<String, Object> extra;
```

Comment:
I find the name extra a bit unconventional, but don't have better one. So ok

Reply:
For clarity/consistency reasons, we keep the same field names as in the database API.